pandas/tools/tests/test_merge.py

```diff
@@ -2255,6 +2255,16 @@ def test_with_mixed_tuples(self):
         df2 = DataFrame({ u'B' : 'foo', (u'B',1) : 'bar' },index=range(2))
         result = concat([df1,df2])
 
+    def test_concat_Nat_series(self):
+        # GH 11693
+        # test for merging NaT series with datetime series.
+        x = pd.Series( pd.date_range('20151124 08:00', '20151124 09:00', freq = '1h', tz = "US/Eastern") )
+        y = pd.Series( pd.date_range('20151124 10:00', '20151124 11:00', freq = '1h', tz = "US/Eastern") )
+        y[:] = pd.NaT
+        xy = pd.concat([x,y])
```

Comment:
not 100% sure that we have all of these tests, so can you add this for no-tz's as well, and both series all-nat (both with and w/o tz's). use a loop to check all 4 conditions.